wrangler-core/src/main/java/io/cdap/wrangler/expression/ELContext.java

```diff
@@ -36,6 +37,23 @@
 public class ELContext implements JexlContext {
   private final Map<String, Object> values = new HashMap<>();
 
+  /**
+   * Context object passed to every expresson evaluation.
+   * All properties of this class are public to ensure they can be accessed with dot(.) format.
+   */
+  public static class Context {
```

Comment:
If this class is accessed by other classes outside this class, why not make it top level class? why should it be nested?